dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumResourceScope.kt

```diff
@@ -207,15 +215,15 @@ internal class RumResourceScope(
             error = ErrorEvent.Error(
                 message = message,
                 source = source.toSchemaSource(),
-                stack = throwable.loggableStackTrace(),
+                stack = throwable?.loggableStackTrace(),
                 isCrash = false,
                 resource = ErrorEvent.Resource(
                     url = url,
                     method = method.toErrorMethod(),
                     statusCode = statusCode ?: 0,
                     provider = resolveErrorProvider()
                 ),
-                type = throwable.javaClass.canonicalName
+                type = throwable?.javaClass?.canonicalName
```

Comment:
Maybe we could have a default type when status code is known : `"HTTP $statusCode"`